src/app/admin/pages/learning-objects/learning-objects.component.scss

```
@@ -7,15 +7,14 @@
   box-sizing: border-box;
   min-height: 500px;
   overflow: hidden;
-  margin-top: 40px;
 }
 
 .list__body {
   .body__column-headers,
   ::ng-deep .row-item {
     padding: 0 20px;
     display: grid;
-    grid-template-columns: 32px 2.5fr 100px 80px 120px 50px;
+    grid-template-columns: 0px 32px 2.5fr 100px 80px 120px 50px;
```

If you use 0px its generally good practice to not append px or another unit of measure, just 0.
```suggestion
    grid-template-columns: 0 32px 2.5fr 100px 80px 120px 50px;
```